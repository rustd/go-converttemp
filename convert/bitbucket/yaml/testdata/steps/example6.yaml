# Example using the size option to increase the memory available to a pipeline step

pipelines:
  default:
    - step:
        size: 2x
        script:
          - echo "This step gets double the memory!"
