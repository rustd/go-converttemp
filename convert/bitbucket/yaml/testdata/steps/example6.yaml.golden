{
    "pipelines": {
        "default": [
            {
                "step": {
                    "size": "2x",
                    "script": [
                        "echo \"This step gets double the memory!\""
                    ]
                }
            }
        ]
    }
}